toolbox/fdc3-conformance/instructions.md

```diff
@@ -0,0 +1,52 @@
+# How to Apply For FDC3 Certification for a Desktop Agent
+
+## 1. Run The Conformance Tests Yourself.
+
+See [Readme](README.md) about how to do this.  
+ 
+## 2. Raise An Issue On This project
+
+[Raise An Issue](https://github.com/finos/FDC3-Conformance-Framework/issues) on the FDC3 Conformance Project.  
+
+In all likelihood, you will need to liase with the maintainers of this project in order to get the conformance tests to work with your desktop agent.  
+
+This will also kick off the process of applying for the "FDC3 Certified" mark.
+
+- Provide details in the issue about how you configured and ran the conformance tests in step 1 above.
+- Provide the test results (number of passes / fails, details of failed tests, versions used) in the issue.
+
+## 3.  Replication
+
+Maintainers of the FDC3 Conformance Framework will aim to replicate your test results.  You must work with them to provide the correct version(s) of your software and resolve any issues around licensing that may arise in order  that they can proceed.  
+
+## 4.  Fees
+
+Currently, in order to grow the standard adoption, certification is provided free of charge until 1st October 2023.  From then, fees for certification will be:
```

Comment:
Is the certification still provided for free? If so, until what date in the future?

If not, we should update this line:
```suggestion
Fees for certification are:
```